website/docs/plugins/ray_launcher.md

```diff
@@ -0,0 +1,165 @@
+...
+[HYDRA] Ray not initialized, init with config: {'num_cpus': 2, 'num_gpus': 0}
+...
+
+```
+
+
+ 
+#### ray_aws launcher
+Override `hydra/launcher` in your config or from command line (shown in example below):
+```yaml
+defaults:
+  - hydra/launcher: ray_aws
+```
+
+`ray_aws` launcher is built on top of ray's [autoscaler cli](https://docs.ray.io/en/latest/autoscaling.html). To get started, you need to 
+config your AWS credentials first, tutorials can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+To verify you've configured AWS correctly, run the following and make sure there's no errors. 
+```commandline
+ python -c 'import boto3;boto3.client("ec2")'
+```
+
+`ray autoscaler` expects a yaml file to provide specs for the new cluster, which we've schematized in `hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`, 
+The plugin defaults are in `conf/hydra/launcher/ray_aws.yaml`. You can override the default values in your app config or from command line.
+
+
+<div class="alert alert--info" role="alert">
+NOTE: To make sure plugin runs as expected, you need to make sure your local machine and your AWS cluster runs the same version of: `ray`, `hydra-core` and `python`. You can 
+easily install the correct software version by overriding `RayClusterConf.setup_commands`
+</div><br/>
+
+Now we can go ahead and run `train.py` using `ray_aws` launcher
+
+```commandline
+$ tree -L 1
+.
+├── conf
+├── model
+└── train.py
+
+$ python train.py hydra/launcher=ray_aws ray_mode=aws random_seed=1,2,3 -m
+...
+[HYDRA] Ray Launcher is launching 3 jobs, 
+[HYDRA]        #0 : ray_mode=aws random_seed=1
+[HYDRA]        #1 : ray_mode=aws random_seed=2
+[HYDRA]        #2 : ray_mode=aws random_seed=3
+...
+(pid=17975) [__main__][INFO] - Start training...
+(pid=17975) [model.my_model][INFO] - Init my model.
+(pid=17976) [__main__][INFO] - Start training...
+(pid=17976) [model.my_model][INFO] - Init my model.
+(pid=17976) [__main__][INFO] - Start training...
+(pid=17976) [model.my_model][INFO] - Init my model. 
+.....
+[HYDRA] Stopped AWS cluster. since you've set your provider.cache_stopped_nodes to be True, we are not deleting the cluster.
+```
+
+In the example app config, we've configured the launcher to download ``*.pt`` files created by the app to local ``download`` dir. You should be able to see a ``download`` dir created in your current working dir.
+
+```commandline
+$ tree -L 1
+.
+├── conf
+├── downloads # Created by example app train.py
+├── model
+├── multirun
+└── train.py
+
+$ tree downloads/
+downloads/
+└── multirun
+    └── 2020-05-18
+        └── 15-17-08
+            ├── 0
+            │   └── checkpoint
+            │       └── checkpoint_1.pt
+            ├── 1
+            │   └── checkpoint
+            │       └── checkpoint_2.pt
+            └── 2
+                └── checkpoint
+                    └── checkpoint_3.pt
+```
+
+
+##### Manage Cluster LifeCycle
```

Comment:
I think it's best to describe the relevant configuration flags instead of trying to provide complete examples.